frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -0,0 +1,90 @@
+import styled from "@emotion/styled";
+import { css } from "@emotion/react";
+import { color } from "metabase/lib/colors";
+import {
+  breakpointMinSmall,
+  breakpointMinLarge,
+  breakpointMinMedium,
+} from "metabase/styled-components/theme";
+
+export const Root = styled.div<{
+  hasScroll: boolean;
+  isBordered?: boolean;
+}>`
+  display: flex;
+  flex-direction: column;
+
+  ${props =>
+    props.hasScroll &&
+    css`
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+    `}
+
+  ${props =>
+    props.isBordered &&
+    css`
+      border: 1px solid ${color("border")};
+      border-radius: 8px;
+      box-shadow: 0 2px 2px ${color("shadow")};
+    `}
+`;
+
+export const ContentContainer = styled.div<{ hasScroll: boolean }>`
+  display: flex;
+  flex-direction: column;
+  flex: 1 0 auto;
+  position: relative;
+
+  ${props =>
+    props.hasScroll &&
+    css`
+      overflow-y: auto;
+    `}
```

Comment:
```suggestion
  overflow-y: ${props => props.hasScroll && "auto"};
```

What do you think about using inline function in the value portion instead when we want to only modify a single property to make it cleaner?